README.md

```diff
@@ -63,6 +64,30 @@ yarn run dev
 
 In this project we are using `yarn` because `netlify` builds our site with `yarn`.
 
+### Folder style guide
```

Comment:
Instead of an entire "Folder style guide" section, I think this would be better under the "Article title" section.